src/Components/Form/AddFormButton.js

```diff
@@ -1,4 +1,4 @@
-import ButtonAdd from '../UI/Button/AddButton.styles';
+import ButtonAdd from '../UI/Form/Button/AddButton.styles';
 
 export default function AddFormButton({ onSetFormButton }) {
 	return (
```

Comment:
This comment is about the next line: what does `prevCount` mean? Is there a counter somewhere? Shouldn't it better be `previousState`?
If you decide to use the `formButton` prop in this component you could also write: `onSetFormButton(!formButton)`

Reply:
It seems you deleted the `AddFormButton` component and decided for a more straight forward approach. Nice!